src/test/java/com/google/roomies/servlets/ListingsDatabaseTest.java

```diff
@@ -101,6 +101,17 @@ public void testAddListingAsMap_listingHasInvalidInput_exceptionThrownBeforeList
     db.addListingAsMap(LISTING_COLLECTION_NAME, listing);
   }
 
+  @Test
+  public void testGetAllDocumentsInCollection_fetchesListingFromFirestore() throws Exception {
+    db.addListingAsMap(LISTING_COLLECTION_NAME, listing);
+    Map<String, Object> expectedData = listing.toMap();
+
+    db.getAllDocumentsInCollection(LISTING_COLLECTION_NAME);
+
+    verify(dbMock, Mockito.times(2)).collection(LISTING_COLLECTION_NAME);
```

Comment:
db and dbMock are a confusing pair of names, since it sounds like you have a real and mocked instance of the same thing. Would generally recommend against abbreviations for something that short anyway. DAMP = Descriptive And Meaningful Phrases is mostly meant for test names, but can apply to variables too.